import_full.py

```diff
@@ -0,0 +1,25 @@
+import listenbrainz_spark
+import listenbrainz_spark.config as config
+import shutil
+import sys
+import tempfile
+import time
+
+from listenbrainz_spark import hdfs_connection
+from listenbrainz_spark.data.import_data import import_dump
+from listenbrainz_spark.data.import_data.ftp import ListenBrainzFTPDownloader
+
+if __name__ == '__main__':
+    if len(sys.argv) != 2:
+        print("Usage: import.py <app_name>")
+        sys.exit(-1)
+
+    listenbrainz_spark.init_spark_session(sys.argv[1])
```

Comment:
We need to catch exceptions and print nice error messages at the top level. And make sure to use good exit codes and practices set forth by https://github.com/metabrainz/metabrainz-howto-guides/blob/master/writing_effective_data_processing_scripts.md